docs/installation/frontend-development.md

```diff
@@ -0,0 +1,111 @@
+# Installation for development
+
+<div class="Alert Alert--warning">
+
+1. First visit [backend installation][be] part to make the application work properly<br>
+2. Use backend URL in your `.env` file or use [cli commend](installation/frontend?id=configuration).
+
+</div>
+
+## Requirements
+---
+
+1. In case you don't have [node.js][node] installed - install the latest stable version.
+2. Choose your package manager:
+   1. [npm][npm]
+   2. [yarn][yarn]
+3. Install [git][git] if needed.
+
+## Quick Start
+---
+If you are an Ergonode Partner implementing **Ergonode** for your clients - this installation is for you:
+
+
+**1) Getting started**
+
+Clone project repository to your local directory:
+
+```bash
+git clone git@github.com:ergonode/skeleton-frontend.git
```

Comment:
isn't the `git@github.com:ergonode/frontend.git` repository meant for development?

Reply:
on the backend repository is the same division.